_posts/2020-07-16-building-drivers-on-zephyr.md

```diff
@@ -0,0 +1,591 @@
+---
+title: "Tips on Building Drivers for Zephyr"
```

Comment:
Maybe: Guide to Writing New Drivers for Zephyr

Your post is more than just tips...it's pretty much a guide

Reply:
changed it to _How to Build Drivers for Zephyr_ How-to's always do well. 👍